### PR TITLE
Google Cloud Storage service

### PR DESCRIPTION
**Pull request by pierreTklein**
Created a service that allows for users to upload, download, and delete files from GCP.

### CODE REVIEW

File: tests/setup.spec.js [hunk 1]
```diff
@@ -18,16 +18,14 @@ const Util = {
 
 //make sure that we are connected to the database
 before(function(done) {
-    this.timeout(0);        
+    this.timeout(60000);
```
Comment (YiFeiZhang2) [inline tests/setup.spec.js]: What is the timeout for?

File: tests/util/bus.test.util.js [hunk 2]
```diff
@@ -59,5 +59,8 @@ function dropAll(callback) {
             logger.error(`could not drop Buses. Error: ${JSON.stringify(err)}`);
             callback(err);
         }
-    );
+    ).catch((error) => {
```
Comment (YiFeiZhang2) [inline tests/util/bus.test.util.js]: I think .then(success, failure) already covers the case of .catch(error).

Comment (pierreTklein) [reply tests/util/bus.test.util.js]: Oh-- hmm.. I was dealing with un-caught promises, and this fixed it.. maybe there's a promise that doesn't have the , failure parameter?

File: tests/setup.spec.js (hunk 1 above)
Comment (pierreTklein) [reply tests/setup.spec.js]: So that there is a timeout at all!

**Pull request merged**